demisto_sdk/commands/test_content/TestContentClasses.py

```diff
@@ -43,6 +43,9 @@
                             "the instance_name argument in conf.json. The options are:\n{}"
 ENTRY_TYPE_ERROR = 4
 DEFAULT_INTERVAL = 4
+MAX_RETRIES = 3
+RETRIES_THRESHOLD = MAX_RETRIES // 2
```

Comment:
```suggestion
RETRIES_THRESHOLD = ceil(MAX_RETRIES / 2)
```
you should also add
`from math import ceil`

Reply:
Done